tests/environment/.env

```diff
@@ -1,9 +1,21 @@
+ALFRESCO_TAG=latest
 SOLR6_TAG=2.0.3
 POSTGRES_TAG=13.3
 ACTIVEMQ_TAG=5.16.1
 AIMS_TAG=1.2
 SYNC_SERVICE_TAG=3.7.0
 ES_TAG=7.10.1
+TRANSFORM_ROUTER_TAG=1.5.2
```

Comment:
Isn't the `TRANSFORM_ROUTER_TAG` dynamically evaluated [here](https://github.com/Alfresco/alfresco-community-repo/blob/master/packaging/tests/scripts/start-compose.sh#L31)?

Reply:
I added it in the .env for local runs without having to use the start-compose script